public/app/features/dashboard/components/PanelEditor/usePanelLatestData.ts

```diff
@@ -63,8 +63,10 @@ export const usePanelLatestData = (
 
   return {
     data: latestData,
-    error: latestData && latestData.error,
```

Comment:
this was only checking the deprecated property... not the other flavors that will show alerts in the panel